cluster/manifests/audittrail-adapter/daemonset.yaml

```diff
@@ -36,7 +36,7 @@ spec:
       hostNetwork: true
       containers:
       - name: audittrail-adapter
-        image: pierone.stups.zalan.do/teapot/audittrail-adapter:master-25
+        image: pierone.stups.zalan.do/teapot/audittrail-adapter:master-26
```

Comment:
What changed in the image?

Reply:
`[AUTOMATED] Update Alpine base image 3.12 -> 3.13`